server/controllers/stores-controller.js

```diff
@@ -1,61 +1,64 @@
-const stores = require('./../data/stores.json')
+const stores = require("./../data/stores.json");
 
-function geoDist(lat1, lon1, lat2, lon2){
+function geoDist(lat1, lon1, lat2, lon2) {
```

Comment:
It would be a good idea to add some documentation on the Math going on here (do in follow up bug as well)